Neos.Flow/Classes/Aop/AdvicesTrait.php

```diff
@@ -52,7 +52,8 @@ public function Flow_Aop_Proxy_invokeJoinPoint(\Neos\Flow\Aop\JoinPointInterface
         if (__CLASS__ !== $joinPoint->getClassName()) {
             return parent::Flow_Aop_Proxy_invokeJoinPoint($joinPoint);
         }
-        if (isset($this->Flow_Aop_Proxy_methodIsInAdviceMode[$joinPoint->getMethodName()])) {
+        $methodName = $joinPoint->getMethodName();
+        if (isset($this->Flow_Aop_Proxy_methodIsInAdviceMode[$methodName])) {
```

Comment:
```suggestion
        if (isset($this->Flow_Aop_Proxy_methodIsInAdviceMode[$joinPoint->getMethodName()])) {
```
Let's maybe not do this here?

Reply:
Right, how did that sneak in

Reply:
I was about to replace the following line with `return self::$methodName(...$joinPoint->getMethodArguments());` but then was unsure and it turned out in a half-change.

Reply:
sure!

Reply:
Note: because the `$methodName` might be a static method (?)